internal/k8s/owner_fetcher.go

```diff
@@ -211,15 +216,20 @@ func (v OwnerFetcher) OwnerTreeOf(ctx context.Context, entity K8sEntity) (result
 	}()
 
 	ref := entity.ToObjectReference()
-	return v.ownerTreeOfHelper(ctx, ref, meta)
+	return v.ownerTreeOfHelper(ctx, ref, meta, nil)
 }
 
-func (v OwnerFetcher) ownerTreeOfHelper(ctx context.Context, ref v1.ObjectReference, meta metav1.Object) (ObjectRefTree, error) {
+func (v OwnerFetcher) ownerTreeOfHelper(ctx context.Context, ref v1.ObjectReference, meta metav1.Object, path []types.UID) (ObjectRefTree, error) {
 	tree := ObjectRefTree{Ref: ref, CreationTimestamp: meta.GetCreationTimestamp()}
 	owners := meta.GetOwnerReferences()
 	for _, owner := range owners {
+		// TODO: Owner references can also exist at cluster scope, for which this incorrectly propagates the parent ref's Namespace.
```

Comment:
Not sure what you want to do here. I noticed this as I was testing but it doesn't seem to materially change how Tilt operates and as far as I could tell there's no easy way in this code path to determine if a resource exists at cluster scope. The namespace of the owner reference is not specified in its object ([ref](https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#OwnerReference)).

> OwnerReference contains enough information to let you identify an owning object. An owning object must be in the same namespace as the dependent, or be cluster-scoped, so there is no namespace field. +structType=atomic

Reply:
maybe it's ok? our client library wrappers all require a namespace argument, but i think they just drop it if the resource is cluster-scoped.

Reply:
Want me to adjust to a `NOTE` or anything?